VMBackup/main/workloadPatch/WorkloadPatch.py

```diff
@@ -421,6 +445,12 @@ def timeoutDaemon(self):
             argsDaemon = ["su", "-", self.linux_user, "-c", preDaemonOracle]
             devnull = open(os.devnull, 'w')
             daemonProcess = subprocess.Popen(argsDaemon, stdout=devnull, stderr=devnull)
+        elif 'postgres' in self.name.lower():
```

Comment:
as discussed, please have a look whether we can keep call this timeoutDaemon as part of a subprocesss ?
So it will be part of infra instead every workload owner writing a timeout script

Reply:
created a common timeoutDaemon shell script which with sleep for timeout and call the corresponding post script